dev/documentation/en/datamodel-keyconcepts.md

```diff
@@ -0,0 +1,100 @@
+---
+book: dev-documentation
+version: 3.4
+title: Entities Key Concepts - Technical Documentation - OJS|OMP|OPS
+---
+
+# Entities: Key Concepts
+
+The overall database schema for the PKP data model consists of well over 100 tables.
+The present page does not introduce the entire data model.
+It merely characterizes a few particularly important entities
+and describes some other things that one might _expect_ to be entities
+but that happen to be represented in some other way.
+
+The page is organized as a loose list of sections in alphabetical order.
+
+When you look for the classes representing these entities,
+some of them will live in PKPlib, others in OJS (etc.),
+and many in both, with an inheritance relationship between them. 
+In the latter case, the superclass in PKPlib usually has a `PKP` name prefix.
+
+## Author
+
+Authors are associated with [Publications](#Publication).
+An Author need not be a [User](#User).
+Except for the email address, an Author's attributes such as
+names and affiliations can be localized.
```

Comment:
When making the distinction between authors and users, we should describe authors as a kind of data record. Users are part of the access and control system of the software. Authors are merely metadata attached to a publication, and may not even represent a person.